CompilerSource/compiler/compile.cpp

```diff
@@ -105,29 +105,28 @@ int lang_CPP::compile(EnigmaStruct *es, const char* exe_filename, int mode)
   ide_dia_clear();
   ide_dia_open();
   cout << "Initialized." << endl;
+  string compilepath = CURRENT_PLATFORM_NAME "/" + extensions::targetOS.builddir + "/" + extensions::targetOS.target;
```

Comment:
We can probably get away with just the current platform name and the target ey file name, but this should be fine.

Reply:
Leave it as is, people using the cross-compiler will have to rebuild less.